lms/djangoapps/discussion_api/tests/test_forms.py

```diff
@@ -100,12 +100,14 @@ def test_empty_topic_id(self):
         self.form_data.setlist("topic_id", ["", "not empty"])
         self.assert_error("topic_id", "This field cannot be empty.")
 
-    def test_following_true(self):
-        self.form_data["following"] = "True"
+    @ddt.data("True", "true", 1, True)
+    def test_following_true(self, value):
+        self.form_data["following"] = value
         self.assert_field_value("following", True)
 
-    def test_following_false(self):
-        self.form_data["following"] = "False"
+    @ddt.data("False", "false", 0, False)
+    def test_following_false(self, value):
```

Comment:
Although not part of the ticket, since we're looking at the test coverage again, is there a `test_invalid_following` method?  That is, is there a unit test for when someone passes an invalid value for `following`?